templates/nextflow-aurora-mysql.yaml

```diff
@@ -103,6 +103,12 @@ Resources:
         PasswordLength: 16
       KmsKeyId: !Sub ${AuroraKeyStack.Outputs.Key}
 
+  AuroraMasterSecretRotationSchedule:
+    Type: AWS::SecretsManager::RotationSchedule
```

Comment:
Doesn't this need to be paired with a Lambda function to update the database accordingly? For example, using the `HostedRotationLambda` property ([docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-secretsmanager-rotationschedule-hostedrotationlambda.html)). I've never configured secret rotation, so I might be completely off-base.

Reply:
you are correct.  The docs says configuring `HostedRotationLambda` was not required so i was hoping that AWS would just figure it out for us.  I've tested and that is indeed not the case.

Reply:
done, i have set to use the AWS built in `MySQLSingleUser` lambda and explicitly set the rotation schedule to 30 days.